kernel/byterun/coq_interp.c

```diff
@@ -1380,14 +1400,16 @@ value coq_interprete
         bigint = *sp++; /* TODO: take accu into account */
         value divisor;
         divisor = *sp++;
-        if (uint63_eq0(divisor)) {
+        int b;
+        Uint63_eq0(b, divisor);
```

Comment:
Are we confident no allocation can kill `divisor` and `bigint` here? It might be better to adjust the stack later.